data/pages-content/pyvo-meetup.md

```diff
@@ -0,0 +1,23 @@
+---
+ title: Pyvo Meetup @ EuroPython
+ subtitle:
+   We're hosting our July Pyvo meetup during EuroPython! Let's grill & chill, connect and share!
+ social_card: ep2023-pyvo-meetup-sc.png
+---  
+
+# Pyvo Meetup
+## About Pyvo
+ “Pivo” means “beer” in Czech, and Pyvo is a meetup for fans of the Python programming language and related technologies. These meetups are held in several cities across the country. The meetups are quite informal, and they're held in pubs usually with talks and mingling.
+## About the Event
+Prague Pyvo is traditionally held on the third Wednesday of each month. The July edition will take place during EuroPython! Join us for our July Pyvo meetup in Prague, where we'll be hosting a delightful grill & chill session. We invite you to connect with the Czech local Python community in Prague, enjoy the perfect summer vibes of Prague and exchange knowledge and ideas!
+
+<div style={{textAlign: "center", marginBottom: 20}}>
+<ButtonLink href="https://www.meetup.com/pyvo-prague-python-meetup/events/nzltctyfckbzb/"> Register your interest now! </ButtonLink>
```

Comment:
```suggestion
<ButtonLink href="https://www.meetup.com/pyvo-prague-python-meetup/events/293781460/"> Register your interest now! </ButtonLink>
```

Reply:
Looks great, thanks a lot! 🙂 I've sent you the wrong link, so I've added one suggestion. I've updated our web and meetup, so I guess we can merge it? 🙂